src/auto4_lua_dialog.cpp

```diff
@@ -286,6 +286,10 @@ namespace Automation4 {
 					max = DBL_MAX;
 					min = -DBL_MAX;
 				}
+				if (step != 0.0) {
+					min = min == -DBL_MAX ? 0.0 : min;
```

Comment:
Does this impact any scripts that you know of seeing significant use? In particular, if it's anything we can fix preemptively in TSTools we should.

Reply:
The ones I know of are Aegisub-Motion (https://github.com/TypesettingTools/Aegisub-Motion/blob/9719ce019a33b2a27a0e874d944e2d476738a63f/Aegisub-Motion.moon#L97) and lyger's Blur Clip (https://github.com/TypesettingTools/lyger-Aegisub-Scripts/blob/caa697513ec22087faaa357cbc89231e7b2e94a3/macros/lyger.ClipBlur.lua#L283).